src/components/HelperErrorText/HelperErrorText.mdx

```diff
@@ -50,40 +50,40 @@ use case of the `HelperErrorText` component is to dynamically render error text
 when a form field is invalid. In that scenario, the dynamically updated error
 text will be read by assistive technologies.
 
-The example below is shows a `TextInput` component. The `TextInput` component
+The example below shows a `TextInput` component. The `TextInput` component
 uses the `HelperErrorText` component to render the red-colored `"This is error text :("` text below the input field and that text element is referenced with
 the `aria-describedby` attribute on the HTML input element. This type of
 association is handled automatically by all DS `Form Elements` components.
 
 <Canvas of={HelperErrorTextStories.TextInputExample} />
 
-### ariaAtomic
+### ARIA Atomic
 
-The `ariaAtomic` prop sets the `aria-atomic` HTML attribute. This allows the
-entire DOM element to be presented to assistive technologies. When it is set to
-`false`, only additions or removals will be read by assistive technologies.
+By default, `aria-atomic` is set to `true` for the `HelperErrorText` component.
+This allows the entire DOM element to be presented to assistive technologies. If
+desired, you can pass `aria-atomic={false}` instead so that only additions or
+removals will be read by assistive technologies.
```

Comment:
I know this is how it was written previously, but can you clearly define what `additions and removals` means?